utils/saasherder.py

```diff
@@ -325,7 +325,7 @@ def _process_template(self, options):
                                 get_commit_sha_options)
                             consolidated_parameters['IMAGE_TAG'] = image_tag
 
-            oc = OC('server', 'token')
+            oc = OC('server', 'token', local=True)
```

Comment:
Can we do `OC(None, None)` and consider is local when server is None?

Reply:
i just don't want to dive all the way into https://github.com/app-sre/qontract-reconcile/blob/542b0bbf1df84871ce6d39eb8abce8db6039fe7b/utils/oc.py#L439

Reply:
Fair enough.